cmake/usage/usage.cmake

```diff
@@ -1,9 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0
-if(NOT DEFINED ENV{ZEPHYR_BASE})
+if(NOT DEFINED {ZEPHYR_BASE})
```

Comment:
Curly braces should be removed here.

Was this a typo, or is there a search and replace that needs to be fixed?

Reply:
guess my sed only did part of the job 😉 
Thanks for spotting.

Reply:
fixed, and while I was looking at `usage.cmake` I discovered some old wrong behavior where `ninja usage` would print:
```
Build flags:

  ninja VERBOSE=1 [targets] verbose build
```
which has been fixed now, of course.
https://github.com/zephyrproject-rtos/zephyr/pull/23054/commits/83b40a3a041df1fea5d8296a21cf41b1577178ba